### PR TITLE
adding ps command

### PR DESCRIPTION
**Pull request by mikebrow**
Here's the start of some code to add a simple ps command to runc to address issue #380 

Thought I'd put this up and ask for comments.

Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

File: ps.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		if factory == nil {
+			err := factoryPreload(context)
+			if err != nil {
+				logrus.Fatal(err)
+				return
+			}
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
```
Comment (hqhq) [inline ps.go]: We can use `--root` to change the state directory for different containers, I think we may need some constant directory for information that `runc ps` needs.

Comment (mikebrow) [inline ps.go]: Or let users that use a different root directory enter the --root option when they run the runc command?

**Comment by mikebrow** [comment]
I could modify the usage instructions to say "lists containers started by runc with the given root"...

**Comment by mikebrow** [comment]
Added a created column to the ps command, similar to the docker ps command created column which indicates how long the container has been running.  Also modified the usage instructions to refer to the fact that the ps is for a given root.

**Comment by mikebrow** [comment]
@duglin suggests showing the time stamp for when the process was started vs the time running information.

**Comment by crosbymichael** [comment]
We should name this `list` and not `ps`

**Comment by mikebrow** [comment]
I'm re-basing this PR right now.  Will change the command to list or wdyt of ls?

**Comment by crosbymichael** [comment]
In the future you can just change your original pr by force pushing to your branch instead of making new prs ;)

**Comment by mikebrow** [comment]
:-) Ok.

**Pull request closed without merging**